app/waf/src/models/adc.model.ts

```diff
@@ -17,6 +17,110 @@
 import {model, property, Entity} from '@loopback/repository';
 import {CommonEntity} from '.';
 
+const adcComputeProps = {
+  imageRef: {
+    type: 'string',
+    format: 'uuid',
+  },
+  flavorRef: {
+    type: 'string',
+    // format: 'uuid',
+  },
+  sshKey: {
+    type: 'string',
+  },
+  userData: {
+    type: 'string',
+  },
+};
+
+const mgmt1Props = {
+  type: {
+    type: 'string',
+    enum: ['mgmt'],
+  },
+  networkId: {
+    type: 'string',
+    format: 'uuid',
+  },
+  fixedIp: {
+    type: 'string',
+    format: 'ipv4',
+  },
+};
+
+const failover1Props = {
+  type: {
+    type: 'string',
+    enum: ['ha'],
+  },
+  networkId: {
+    type: 'string',
+    format: 'uuid',
+  },
+  fixedIp: {
+    type: 'string',
+    format: 'ipv4',
+  },
+};
+
+const internal1Props = {
+  type: {
+    type: 'string',
+    enum: ['int'],
+  },
+  networkId: {
+    type: 'string',
+    format: 'uuid',
+  },
+  fixedIp: {
+    type: 'string',
+    format: 'ipv4',
+  },
+};
+
+const external2Props = {
+  type: {
+    type: 'string',
+    enum: ['ext'],
+  },
+  networkId: {
+    type: 'string',
+    format: 'uuid',
+  },
+  fixedIp: {
+    type: 'string',
+    format: 'ipv4',
+  },
+  floatingIp: {
+    type: 'string',
+    format: 'ipv4',
+  },
+};
+
+const adcNetworksProps = {
```

Comment:
The 'networks' property is a dictionary, so it allows user to input any key. For instance:

```
{
  "networks" {
    "mgmtABC" : {},
    "ha": {},
    "internal123": {},
    "external456" {}
  }
}
```

We can not type the exact key name in API schema, and should use another way to validate this dictionary.
https://swagger.io/docs/specification/data-models/dictionaries/